.env.example

```
@@ -1,13 +1,50 @@
+
+# docker 기반 배포용
+REDIS_HOST=redis
+
 
 # JWT — 256비트 이상 랜덤 문자열 권장
-JWT_SECRET=your-very-long-secret-key-here
-JWT_ACCESS_EXPIRATION=1800000
-JWT_REFRESH_EXPIRATION=604800000
\ No newline at end of file
+JWT_SECRET=0exyGqsGgbtyx36bStLrM38/XzzGSgcrv2RCC0b6ylk=
+JWT_ACCESS_TOKEN_EXPIRATION=1800000
```

dotenv-linter (4.0.0)

[warning] [UnorderedKey] The JWT_ACCESS_TOKEN_EXPIRATION key should go before the JWT_SECRET key

(UnorderedKey)